packages/@aws-cdk/aws-servicecatalog/test/integ.portfolio.ts

```diff
@@ -0,0 +1,25 @@
+import * as iam from '@aws-cdk/aws-iam';
+import { App, Stack } from '@aws-cdk/core';
+import { Portfolio } from '../lib';
```

Comment:
Use the qualified import here too:

```suggestion
import * as servicecatalog from '../lib';
```

This validates everything that is needed is exported by the module.

Reply:
Still not addressed.

Reply:
missed this one somehow